src/main/bootstrap/mod.rs

```diff
@@ -332,6 +330,18 @@ fn shuffled_bootstrap_peers(
     peers
 }
 
+fn update_peers(peers: &mut Vec<PeerInfo>, rx: Receiver<Vec<PeerInfo>>) {
```

Comment:
The name is very generic, maybe `receive_peers_into()` or some docs about what the function does? :)